### PR TITLE
fixing links in ahref and anohref

### LINKED ISSUE #141: Style changes on table: Rules of ARIA attributes usage by HTML language feature
With recent updates to the Editor's Draft there seems to be some styling changes to the table for Rules of ARIA attributes usage by HTML language feature. 
The currently published Working Draft has grey table headings and is striped white and grey. Columns are also separated.
The Editor's Draft uses dark blue headings and is striped blue and grey. Columns are not visually separated. A class="simple" has been applied to the table.

I'm wondering if this styling change is intentional or is it a side effect of the recent updates?

### LINKED ISSUE #142: id references for a and href have gone missing
The editor's draft seems to be missing a couple of references for a with and without href.

a element with a href currently:
`<a><code>a</code></a> element with a <code>href</code>`

possible change to:
`<a><code>a</code></a> element with a <a data-cite="html/links.html#attr-hyperlink-href"><code>href</code></a>`

a element without a href currently:
`<code>a</code></a> element without a <code>href</code>`

possible change to:
`<a><code>a</code></a> element without a <a data-cite="html/links.html#attr-hyperlink-href"><code>href</code></a>`

### PR DESCRIPTION
**Pull request by pgrucza**
This closes #141 and closes #142

applied changes as suggested in #142

***
<a href="https://pr-preview.s3.amazonaws.com/pgrucza/html-aria/pull/143.html" title="Last updated on May 31, 2019, 3:36 PM UTC (5aa6df2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/html-aria/143/ec286f4...pgrucza:5aa6df2.html" title="Last updated on May 31, 2019, 3:36 PM UTC (5aa6df2)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -152,7 +152,7 @@ <h2 id="docconformance">
         </tr>
         <tr id="ahref" tabindex="-1">
           <td>
-          <a><code>a</code></a> element with a <code>href</code></td>
+          <a><code>a</code></a> element with a <a data-cite="html/links.html#attr-hyperlink-href"><code>href</code></a></td>
```
Comment (marcoscaceres) [inline index.html:155]:
```suggestion
          <a>`a`</a> element with a <a data-cite="html/links.html#attr-hyperlink-href">`href`</a></td>
```

File: index.html [hunk 2]
```diff
@@ -187,7 +187,7 @@ <h2 id="docconformance">
         </tr>
         <tr id="anohref" tabindex="-1">
           <td>
-            <code>a</code></a> element without a <code>href</code>
+            <a><code>a</code></a> element without a <a data-cite="html/links.html#attr-hyperlink-href"><code>href</code></a>
```
Comment (marcoscaceres) [inline index.html:190]:
```suggestion
            <a>`a`</a> element without a <a data-cite="html/links.html#attr-hyperlink-href">`href`</a>
```

**Review by marcoscaceres — APPROVED** [review]

**Comment by marcoscaceres** [comment]
Marked as non substantive for IPR from ash-nazg.

File: index.html (hunk 1 above)
Comment (pgrucza) [reply index.html:155]: @marcoscaceres you seem to be suggesting the removal of the code elements and replacing them with markdown.  The code element is being used in the rest of the document. 

If this is part of a larger change to the markup, I'm not sure if this is the best place to start it.

Comment (marcoscaceres) [reply index.html:155]: Yeah, we can do those in a new pull request.

**Review by marcoscaceres — CHANGES_REQUESTED** [review]
See my comment in the bug about the "simple" class.

**Comment by marcoscaceres** [comment]
@pgrucza, you meed to accept my suggestions before I can merge this.

**Comment by marcoscaceres** [comment]
Actually, will make the code changes independently.

**Comment by pgrucza** [comment]
This closes #141 and closes #142

**Pull request merged**